README.md

```diff
@@ -536,9 +529,39 @@ This allows multiple configs to be generated, for example for different
 applications, and then using copies of a bare OS image of a particular version
 to prepare images for each of those apps.
 
-... Instructions on `config generate` and `os config` ...
+You can generate independent configuration files using the
+`balena config generate` command. This works in a similar way to the
+`balena os configure` command should only an OS version along with an
+application name or device UUID be specified, in that it will interactively ask
+for more details:
```

Comment:
I found this sentence a bit hard to read because of the long part between "should only" and "be specified". Maybe adding a comma after "command" and replacing "should only" with "in case" (because it intially sounded like "command should" went together...).

Reply:
Yep, shall split it up. Cheers. :)